vega_sim/environment/agent.py

```diff
@@ -67,12 +88,31 @@ def initialise(self, vega: VegaService, create_wallet: bool = True):
         else:
             self.vega.login(name=self.wallet_name, passphrase=self.wallet_pass)
 
+    def _update_market_params(self):
+
+        if self.market_params_update_freq is None:
+            return
+
+        if self.step_count % self.market_params_update_freq == 0:
```

Comment:
You can drop the above check and just do `if self.market_params_update_freq and self.step_count % self.market_params_update_freq == 0`